### DISCUSSION TITLE
TVOS Button (CardButtonStyle)

### DISCUSSION THREAD

**Discussion by pnewell** [siteline/SwiftUI-Introspect#295] [Ideas]
Could anyone help me customize introspect to work with a button on TVOS? Based on the docs I suspect a button with CardButtonStyle on TVOS is either a TVCardView or TVLockupView, but my initial attempts haven't worked

**Comment by davdroman** [comment]
@pnewell thanks for raising this, I wasn't aware of these TV view types. I'll look into it and hopefully make it work.

**Comment by davdroman** [comment]
After looking into this, `CardButtonStyle` doesn't render any public TVUIKit components. Instead it renders some subclass of the private `_UIFloatingContentView` class. So there's no view type that can be meaningfully introspected here unfortunately.

<img width="1072" alt="Screenshot 2023-07-05 at 21 52 56" src="https://github.com/siteline/SwiftUI-Introspect/assets/2538074/f8b95216-6951-470a-ba6c-17793d938063">
